docs/features.md

```
@@ -0,0 +1,77 @@
+
+<!-- The TOC below is auto-generated. To regenerate, run: npx markdown-toc -i docs/features.md --maxdepth 3 -->
+<!-- toc -->
+<!-- tocstop -->
+
+---
+
+## Code Completion (with code snippet support)
+
+Press `Ctrl+Space` to see the completion items list, which depends on your cursor position. Code completion items also pop up automatically based on your language-server-specific trigger characters.
+
+![](../resources/images/lsp4intellij-completion.gif)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)